data/korath/korath ships.txt

```diff
@@ -387,6 +387,164 @@ ship "'olofez"
+		shields 43000
+		hull 27000
+		"required crew" 523
+		"bunks" 1433
+		"mass" 1660
+		"drag" 27.6
+		"heat dissipation" .36
+		"fuel capacity" 700
+		"cargo space" 351
+		"outfit space" 856
+		"weapon capacity" 237
+		"engine capacity" 195
+		"capture defense" 200
+		illegal 40000000
+		weapon
+			"blast radius" 600
+			"shield damage" 9000
+			"hull damage" 4800
+			"hit force" 20000
+	outfits
+		"Banisher Grav-Turret"
+		"Warder Anti-Missile" 2
+		"Langrage Hyper-Heaver" 2
+		"Firelight Missile Bank"
+		"Firelight Missile" 20
+
+		"Double Plasma Core"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor" 2
+		"Thermal Repeater Rifle" 526
+		"Microbot Defense Station" 25
+
+		"Afterburner (Planetary Class)"
+		"Reverser (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+		"Jump Drive"
+
+	"reverse engine" -79 -186
+		zoom 0.8
+	"reverse engine" 79 -186
+		zoom 0.8
+	engine -101 196
+		zoom 0.9
+	engine 101 196
+		zoom 0.9
+	engine -60 190 0.8
+	engine 60 190 0.8
+	engine -29 212
+	engine 29 212
+	gun -81.5 58 "Firelight Missile Bank"
+		angle 270
+	turret -96.5 -129.5 "Langrage Hyper-Heaver"
+	turret 96.5 -129.5 "Warder Anti-Missile"
+	turret -62.5 55 "Warder Anti-Missile"
+	turret 62.5 55 "Langrage Hyper-Heaver"
+	turret 36.5 -55.5 "Banisher Grav-Turret"
+	bay "Fighter" -115 -152
+		angle 30
+	bay "Fighter" 115 -152
+		angle 330
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Although fierce in battle and highly adaptable, Korath do not easily recover from injury. Without the aid of a surgeon or more equipment than fits into a standard escape pod, a stranded, injured Korath faces poor odds of survival. Therefore, Korath who leave their home territory have placed great importance on the construction of hospital ships that can quickly locate, recover, and treat the survivors of wrecked ships."
+	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have at hand. Even incubators and nurseries are available to treat the young who grow up aboard ship."
+
+
+ship "Ikatila'ej"
+	sprite "ship/ikatila-ej"
+	thumbnail "thumbnail/ikatila-ej"
+	attributes
+		category "Utility"
+		"cost" 109375000
+		"shields" 72700
+		"hull" 93000
+		"required crew" 957
+		"bunks" 1933
+		"mass" 5380
+		"drag" 58
+		"heat dissipation" .29
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1314
+		"weapon capacity" 440
+		"engine capacity" 259
+		"spinal mount" 1
+		"capture defense" 800
+		illegal 40000000
```

Comment:
Does the code for fining on scan actually check the base attributes of the ship or is it just the outfits?

Reply:
With similar large crew, and me placing 26 Stations on it (more than most I feel), it should be notably hard to capture I think. And since these are rare, I think that should be good

Reply:
Just make the gun illegal if we want to I'd say

Reply:
Also its hard to capture only if you dont have the other ship
If you do you can get this one without too much trouble, the microbot count could be invcreased by 14 easily

Reply:
I think it's fine as is for now, and we can rely on future QoL outfits (or buff MDS's) to fix it. Derpy doesn't want us to increase the outfit space much further.